AmplitudeServiceKitTests/AmplitudeServiceKitTests.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  AmplitudeServiceKitTests.swift
+//  AmplitudeServiceKitTests
+//
+//  Created by Darin Krauss on 6/18/19.
+//  Copyright © 2019 LoopKit Authors. All rights reserved.
+//
+
+import XCTest
+@testable import AmplitudeServiceKit
+
+class AmplitudeServiceKitTests: XCTestCase {
```

Comment:
Should this be committed?

Reply:
Probably not. Will remove the test file itself, but leave the project.